### PR TITLE
feat(chips): Add setAttr adapter method

### PR DESCRIPTION
**Pull request by patrickrodee**
Add `MDCChipAdapter#setAttr` method. Update screenshot tests with appropriate roles.

BREAKING CHANGE: Add the `setAttr` method to the chip adapter.

### CODE REVIEW

File: test/screenshot/spec/mdc-chips/classes/choice.html [hunk 1]
```diff
@@ -44,33 +44,33 @@
...
+            <button class="mdc-chip" role="radio" aria-checked="false">
               <span class="mdc-chip__text">Chip Four</span>
             </button>
           </div>
         </div>
 
         <div class="test-cell test-cell--chip">
           <div class="mdc-chip-set mdc-chip-set--choice">
-            <button class="mdc-chip mdc-chip--selected">
+            <button class="mdc-chip mdc-chip--selected" role="radio" aria-checked="true">
```
Comment (abhiomkar) [inline test/screenshot/spec/mdc-chips/classes/choice.html]: If this is a radio type selection update this demo to have single selection.

Comment (patrickrodee) [reply test/screenshot/spec/mdc-chips/classes/choice.html]: Choice chips are single selection by default. I don't think there's any change to be done for that.

Comment (abhiomkar) [reply test/screenshot/spec/mdc-chips/classes/choice.html]: Right. All radio buttons have `aria-checked="true"` which is not intended for radio group. Am I missing something?

Comment (patrickrodee) [reply test/screenshot/spec/mdc-chips/classes/choice.html]: Maybe I'm missing something. Can you make a commit or open a PR with the change you want? I'm not following.

**Pull request merged**